app/src/main/kotlin/org/eclipse/kuksa/testapp/databroker/view/DataBrokerConnectionView.kt

```diff
@@ -238,7 +236,16 @@ fun DataBrokerConnection(viewModel: ConnectionViewModel) {
 
 @Preview
 @Composable
-fun DataBrokerConnectionPreview() {
+fun DataBrokerConnectionPreview_Disconnected() {
```

Comment:
In fact it is pascal cased. The special case with the "_Disconnected" isn't covered by your link, so no clue if you want to have it "_disconnected". Please make a suggestion here?

Reply:
"DataBrokerConnectionPreviewDisconnected" would be my expectations. "_" is only allowed for test cases. Maybe I am missing something here?

Reply:
Or maybe even better DataBrokerConnectionDisconnectedPreview

Reply:
I don't have a hard preference here, but personally I don't like the "in-between-changing" of the name (DataBrokerConnectionDisconnectedPreview, DataBrokerConnectionConnectedPreview, DataBrokerConnectionConnectingPreview....). It feels like it adds additional complexity because I don't see on first glance what the important information is hidden inside that name. So as long as it is not forbidden by lint or compose I would prefer to keep it like this.

Reply:
Hmm maybe just DisconnectedPreview / ConnectingPreview would be enough because the Context of the "DataBrokerConnection" is already clear from the class name. And then it is easy to read? The _ just feels so off and inconsistent to me. Still wondering why lint is not complaining here.^^

Here are also some more official examples of the naming.
https://developer.android.com/jetpack/compose/tooling/previews#preview-interactive

Reply:
<3